pyopia/cli.py

```diff
@@ -27,6 +29,22 @@
 app = typer.Typer()
 
 
+def get_custom_progress_bar(description, disable):
```

Comment:
suggest moving non-cli functions to the bottom of the file to keep things tidy (together with setup_logging and check_chunks)